src/ronin/profile/Profile.sol

```diff
@@ -71,6 +77,44 @@ contract Profile is IProfile, ProfileXComponents, Initializable {
     emit ProfileMigrated(id, candidateAdmin, treasury);
   }
 
+  /**
+   * @inheritdoc IProfile
+   */
+  function getId2RollupId(
+    address id
+  ) public view returns (uint32) {
+    return _id2Profile[id].rollupId;
+  }
+
+  /**
+   * @inheritdoc IProfile
+   */
+  function getRollupId2Id(
+    uint32 rollupId
+  ) external view returns (address) {
+    (bool found, address id) = _tryGetRollupId2Id(rollupId);
+    if (!found) revert ErrLookUpIdFromRollupIdFailed(rollupId);
```

Comment:
Why dont we just return address zero when not found? Reverting in external view function doesn't make sense

Reply:
we have try function for that. This protects order contracts from querying null data

Reply:
Which other contracts? And should we revert when not found in both methods `getId2Aggregator` and `getId2Sequencer` too?